pyswmm/swmm5.py

```diff
@@ -444,7 +450,10 @@ def getSimulationDateTime(self, timeType):
         errcode = self.SWMMlibobj.swmm_getSimulationDateTime(
             ctypes.c_int(timeType), dtme)
         self._error_check(errcode)
-        return datetime.strptime(dtme.value.decode("utf-8"), "%b-%d-%Y %H:%M:%S")
+        if self.swmm_getVersion() < distutils.version.StrictVersion('5.1.11'):
```

Comment:
`'5.1.11'` this number should go somewhere else, it feels like hardcoding stuff in a "hard to find" place